<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Richard J. Acton" />
  <title>The Genomic loci of specific Human tRNA genes Exhibit Ageing-Related DNA Hypermethylation</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="tRNA-presentation-2019-12-06_files/reveal.js-3.3.0.1/css/reveal.css"/>



<link rel="stylesheet" href="tRNA-presentation-2019-12-06_files/reveal.js-3.3.0.1/css/theme/night.css" id="theme">

<style type="text/css">
.reveal section img {
  background: rgba(255, 255, 255, 0.85);
}
</style>

  <!-- some tweaks to reveal css -->
  <style type="text/css">
    .reveal h1 { font-size: 2.0em; }
    .reveal h2 { font-size: 1.5em;  }
    .reveal h3 { font-size: 1.25em;	}
    .reveal h4 { font-size: 1em;	}

    .reveal .slides>section,
    .reveal .slides>section>section {
      padding: 0px 0px;
    }



    .reveal table {
      border-width: 1px;
      border-spacing: 2px;
      border-style: dotted;
      border-color: gray;
      border-collapse: collapse;
      font-size: 0.7em;
    }

    .reveal table th {
      border-width: 1px;
      padding-left: 10px;
      padding-right: 25px;
      font-weight: bold;
      border-style: dotted;
      border-color: gray;
    }

    .reveal table td {
      border-width: 1px;
      padding-left: 10px;
      padding-right: 25px;
      border-style: dotted;
      border-color: gray;
    }


  </style>

    <style type="text/css">code{white-space: pre;}</style>

    <link rel="stylesheet" href="custom.css"/>

<!-- Printing and PDF exports -->
<script id="paper-css" type="application/dynamic-css">

/* Default Print Stylesheet Template
   by Rob Glazebrook of CSSnewbie.com
   Last Updated: June 4, 2008

   Feel free (nay, compelled) to edit, append, and
   manipulate this file as you see fit. */


@media print {

	/* SECTION 1: Set default width, margin, float, and
	   background. This prevents elements from extending
	   beyond the edge of the printed page, and prevents
	   unnecessary background images from printing */
	html {
		background: #fff;
		width: auto;
		height: auto;
		overflow: visible;
	}
	body {
		background: #fff;
		font-size: 20pt;
		width: auto;
		height: auto;
		border: 0;
		margin: 0 5%;
		padding: 0;
		overflow: visible;
		float: none !important;
	}

	/* SECTION 2: Remove any elements not needed in print.
	   This would include navigation, ads, sidebars, etc. */
	.nestedarrow,
	.controls,
	.fork-reveal,
	.share-reveal,
	.state-background,
	.reveal .progress,
	.reveal .backgrounds {
		display: none !important;
	}

	/* SECTION 3: Set body font face, size, and color.
	   Consider using a serif font for readability. */
	body, p, td, li, div {
		font-size: 20pt!important;
		font-family: Georgia, "Times New Roman", Times, serif !important;
		color: #000;
	}

	/* SECTION 4: Set heading font face, sizes, and color.
	   Differentiate your headings from your body text.
	   Perhaps use a large sans-serif for distinction. */
	h1,h2,h3,h4,h5,h6 {
		color: #000!important;
		height: auto;
		line-height: normal;
		font-family: Georgia, "Times New Roman", Times, serif !important;
		text-shadow: 0 0 0 #000 !important;
		text-align: left;
		letter-spacing: normal;
	}
	/* Need to reduce the size of the fonts for printing */
	h1 { font-size: 28pt !important;  }
	h2 { font-size: 24pt !important; }
	h3 { font-size: 22pt !important; }
	h4 { font-size: 22pt !important; font-variant: small-caps; }
	h5 { font-size: 21pt !important; }
	h6 { font-size: 20pt !important; font-style: italic; }

	/* SECTION 5: Make hyperlinks more usable.
	   Ensure links are underlined, and consider appending
	   the URL to the end of the link for usability. */
	a:link,
	a:visited {
		color: #000 !important;
		font-weight: bold;
		text-decoration: underline;
	}
	/*
	.reveal a:link:after,
	.reveal a:visited:after {
		content: " (" attr(href) ") ";
		color: #222 !important;
		font-size: 90%;
	}
	*/


	/* SECTION 6: more reveal.js specific additions by @skypanther */
	ul, ol, div, p {
		visibility: visible;
		position: static;
		width: auto;
		height: auto;
		display: block;
		overflow: visible;
		margin: 0;
		text-align: left !important;
	}
	.reveal pre,
	.reveal table {
		margin-left: 0;
		margin-right: 0;
	}
	.reveal pre code {
		padding: 20px;
		border: 1px solid #ddd;
	}
	.reveal blockquote {
		margin: 20px 0;
	}
	.reveal .slides {
		position: static !important;
		width: auto !important;
		height: auto !important;

		left: 0 !important;
		top: 0 !important;
		margin-left: 0 !important;
		margin-top: 0 !important;
		padding: 0 !important;
		zoom: 1 !important;

		overflow: visible !important;
		display: block !important;

		text-align: left !important;
		-webkit-perspective: none;
		   -moz-perspective: none;
		    -ms-perspective: none;
		        perspective: none;

		-webkit-perspective-origin: 50% 50%;
		   -moz-perspective-origin: 50% 50%;
		    -ms-perspective-origin: 50% 50%;
		        perspective-origin: 50% 50%;
	}
	.reveal .slides section {
		visibility: visible !important;
		position: static !important;
		width: auto !important;
		height: auto !important;
		display: block !important;
		overflow: visible !important;

		left: 0 !important;
		top: 0 !important;
		margin-left: 0 !important;
		margin-top: 0 !important;
		padding: 60px 20px !important;
		z-index: auto !important;

		opacity: 1 !important;

		page-break-after: always !important;

		-webkit-transform-style: flat !important;
		   -moz-transform-style: flat !important;
		    -ms-transform-style: flat !important;
		        transform-style: flat !important;

		-webkit-transform: none !important;
		   -moz-transform: none !important;
		    -ms-transform: none !important;
		        transform: none !important;

		-webkit-transition: none !important;
		   -moz-transition: none !important;
		    -ms-transition: none !important;
		        transition: none !important;
	}
	.reveal .slides section.stack {
		padding: 0 !important;
	}
	.reveal section:last-of-type {
		page-break-after: avoid !important;
	}
	.reveal section .fragment {
		opacity: 1 !important;
		visibility: visible !important;

		-webkit-transform: none !important;
		   -moz-transform: none !important;
		    -ms-transform: none !important;
		        transform: none !important;
	}
	.reveal section img {
		display: block;
		margin: 15px 0px;
		background: rgba(255,255,255,1);
		border: 1px solid #666;
		box-shadow: none;
	}

	.reveal section small {
		font-size: 0.8em;
	}

}  
</script>


<script id="pdf-css" type="application/dynamic-css">
    
/**
 * This stylesheet is used to print reveal.js
 * presentations to PDF.
 *
 * https://github.com/hakimel/reveal.js#pdf-export
 */

* {
	-webkit-print-color-adjust: exact;
}

body {
	margin: 0 auto !important;
	border: 0;
	padding: 0;
	float: none !important;
	overflow: visible;
}

html {
	width: 100%;
	height: 100%;
	overflow: visible;
}

/* Remove any elements not needed in print. */
.nestedarrow,
.reveal .controls,
.reveal .progress,
.reveal .playback,
.reveal.overview,
.fork-reveal,
.share-reveal,
.state-background {
	display: none !important;
}

h1, h2, h3, h4, h5, h6 {
	text-shadow: 0 0 0 #000 !important;
}

.reveal pre code {
	overflow: hidden !important;
	font-family: Courier, 'Courier New', monospace !important;
}

ul, ol, div, p {
	visibility: visible;
	position: static;
	width: auto;
	height: auto;
	display: block;
	overflow: visible;
	margin: auto;
}
.reveal {
	width: auto !important;
	height: auto !important;
	overflow: hidden !important;
}
.reveal .slides {
	position: static;
	width: 100%;
	height: auto;

	left: auto;
	top: auto;
	margin: 0 !important;
	padding: 0 !important;

	overflow: visible;
	display: block;

	-webkit-perspective: none;
	   -moz-perspective: none;
	    -ms-perspective: none;
	        perspective: none;

	-webkit-perspective-origin: 50% 50%; /* there isn't a none/auto value but 50-50 is the default */
	   -moz-perspective-origin: 50% 50%;
	    -ms-perspective-origin: 50% 50%;
	        perspective-origin: 50% 50%;
}

.reveal .slides section {
	page-break-after: always !important;

	visibility: visible !important;
	position: relative !important;
	display: block !important;
	position: relative !important;

	margin: 0 !important;
	padding: 0 !important;
	box-sizing: border-box !important;
	min-height: 1px;

	opacity: 1 !important;

	-webkit-transform-style: flat !important;
	   -moz-transform-style: flat !important;
	    -ms-transform-style: flat !important;
	        transform-style: flat !important;

	-webkit-transform: none !important;
	   -moz-transform: none !important;
	    -ms-transform: none !important;
	        transform: none !important;
}

.reveal section.stack {
	margin: 0 !important;
	padding: 0 !important;
	page-break-after: avoid !important;
	height: auto !important;
	min-height: auto !important;
}

.reveal img {
	box-shadow: none;
}

.reveal .roll {
	overflow: visible;
	line-height: 1em;
}

/* Slide backgrounds are placed inside of their slide when exporting to PDF */
.reveal section .slide-background {
	display: block !important;
	position: absolute;
	top: 0;
	left: 0;
	width: 100%;
	z-index: -1;
}

/* All elements should be above the slide-background */
.reveal section>* {
	position: relative;
	z-index: 1;
}

/* Display slide speaker notes when 'showNotes' is enabled */
.reveal .speaker-notes-pdf {
	display: block;
	width: 100%;
	max-height: none;
	left: auto;
	top: auto;
	z-index: 100;
}

/* Display slide numbers when 'slideNumber' is enabled */
.reveal .slide-number-pdf {
	display: block;
	position: absolute;
	font-size: 14px;
}

</script>


<script>
var style = document.createElement( 'style' );
style.type = 'text/css';
var style_script_id = window.location.search.match( /print-pdf/gi ) ? 'pdf-css' : 'paper-css';
var style_script = document.getElementById(style_script_id).text;
style.innerHTML = style_script;
document.getElementsByTagName('head')[0].appendChild(style);
</script>

    <script src="tRNA-presentation-2019-12-06_files/htmlwidgets-1.3/htmlwidgets.js"></script>
    <script src="tRNA-presentation-2019-12-06_files/plotly-binding-4.9.0/plotly.js"></script>
    <script src="tRNA-presentation-2019-12-06_files/typedarray-0.1/typedarray.min.js"></script>
    <script src="tRNA-presentation-2019-12-06_files/jquery-1.11.3/jquery.min.js"></script>
    <link href="tRNA-presentation-2019-12-06_files/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
    <script src="tRNA-presentation-2019-12-06_files/crosstalk-1.0.0/js/crosstalk.min.js"></script>
    <link href="tRNA-presentation-2019-12-06_files/plotly-htmlwidgets-css-1.46.1/plotly-htmlwidgets.css" rel="stylesheet" />
    <script src="tRNA-presentation-2019-12-06_files/plotly-main-1.46.1/plotly-latest.min.js"></script>
    <script src="tRNA-presentation-2019-12-06_files/kePrint-0.0.1/kePrint.js"></script>
</head>
<body>
  <div class="reveal">
    <div class="slides">

<section>
    <h1 class="title">The Genomic loci of specific Human tRNA genes Exhibit Ageing-Related DNA Hypermethylation</h1>
    <h2 class="author">Richard J. Acton</h2>
</section>

<section><section id="ageing" class="title-slide slide level1"><h1>Ageing</h1></section><section id="burden-of-age-related-disease" class="slide level2">
<h2>Burden of Age-related Disease</h2>
<p>Ageing is the major risk factor for</p>
<ul>
<li>Cancer</li>
<li>Heart Disease</li>
<li>Dementia</li>
<li>Type 2 Diabetes</li>
<li>Other Chronic Diseases</li>
</ul>
<p><span class="math inline">\(\therefore\)</span></p>
<p>Understanding Ageing</p>
<ul>
<li><span class="math inline">\(\uparrow\)</span> Healthspan / <span class="math inline">\(\downarrow\)</span> Morbidity</li>
<li><span class="math inline">\(\downarrow\)</span> Age-related Diseases</li>
</ul>
<section data-notes="
    - Healthspan - Living longer but chronic conditions
    - Eliminating any one of these minimal impact on average lifespan
    - Frailty ~
">
</section>
</section><section id="multisystem-nature-of-ageing" class="slide level2">
<h2>Multisystem Nature of Ageing</h2>
<p>Epigenetics - Hub of the Hallmarks of Ageing</p>
<p><img src="graphics/Booth2016_F3_epigenomeHubAgingHallmarks.png" width="50%" style="display: block; margin: auto;" /></p>
<p><small>Booth &amp; Brunet 2016 <em>MolCell</em> <span class="citation" data-cites="Booth2016">[<a href="#/ref-Booth2016" role="doc-biblioref">1</a>]</span></small></p>
<p>Loss of epigenetic information <span class="math inline">\(\rightarrow\)</span> Dysregulation</p>
<section data-notes="
    - loss of epigenetic information -&gt; Dysregulation
">
</section>
</section><section id="dna-methylation-changes-with-age" class="slide level2">
<h2>DNA methylation changes with Age</h2>
<ul>
<li>1,000s of Loci Affected</li>
<li>Global Hypomethylation
<ul>
<li>Demethylation of Repetitive elements</li>
<li>Some regions hypermethylating</li>
</ul></li>
<li>Increased Variability</li>
<li>Changes enriched in functional elements
<ul>
<li>Enhancers &amp; Promoters</li>
</ul></li>
</ul>
</section><section id="epigenetic-clocks" class="slide level2">
<h2>Epigenetic Clocks</h2>
<p>Epigenetic Changes Capture Biological Ageing</p>
<p><img src="graphics/LBC1921_Horvath1_survival.png" width="40%" style="display: block; margin: auto;" /></p>
<p>Survival is better in those with decelerated DNAm Age</p>
<p><small>Horvath 2013 <em>Genome Biol</em> <span class="citation" data-cites="Horvath2013">[<a href="#/ref-Horvath2013" role="doc-biblioref">2</a>]</span>, Marioni et al.¬†2015 <em>Genome Biol</em> <span class="citation" data-cites="Marioni2015">[<a href="#/ref-Marioni2015" role="doc-biblioref">3</a>]</span></small></p>
</section></section>
<section id="why-transfer-rnas-trnas" class="title-slide slide level1"><h1>Why Transfer RNAs (tRNAs)?</h1><ul>
<li>Bell et al.¬†2016 <em>Genome Biol</em> <span class="citation" data-cites="Bell2016">[<a href="#/ref-Bell2016" role="doc-biblioref">4</a>]</span></li>
<li>71 age differentially methylated regions
<ul>
<li>39 <span class="math inline">\(\uparrow\)</span>, 32 <span class="math inline">\(\downarrow\)</span></li>
</ul></li>
<li>tRNA-iMet-CAT-1-4 DNAm <span class="math inline">\(\uparrow\)</span> with Age</li>
</ul>
<p><img src="graphics/Bell2016_GenBio_FigS1p55_tRNAMet_a-DMR.png" width="90%" style="display: block; margin: auto;" /></p></section>
<section><section id="why-transfer-rnas-trnas-1" class="title-slide slide level1" data-transition="none"><h1>Why Transfer RNAs (tRNAs)?</h1><ul>
<li>Bell et al.¬†2016 <em>Genome Biol</em> <span class="citation" data-cites="Bell2016">[<a href="#/ref-Bell2016" role="doc-biblioref">4</a>]</span></li>
<li>71 age differentially methylated regions
<ul>
<li>39 <span class="math inline">\(\uparrow\)</span>, 32 <span class="math inline">\(\downarrow\)</span></li>
</ul></li>
<li>tRNA-iMet-CAT-1-4 DNAm <span class="math inline">\(\uparrow\)</span> with Age</li>
</ul>
<p><img src="graphics/Bell2016_GenBio_FigS1p55_tRNAMet_a-DMR_circle.png" width="90%" style="display: block; margin: auto;" /></p></section><section id="trnas" class="slide level2">
<h2>tRNAs</h2>
<p><img src="graphics/translation5.gif" width="50%" style="display: block; margin: auto;" /></p>
<p>Fundamental to the information flow from DNA to Protein</p>
</section><section id="extra-translatory-role" class="slide level2">
<h2>Extra-translatory Role</h2>
<p><img src="graphics/tRNAderivedSmallRNARoles.png" width="55%" style="display: block; margin: auto;" /></p>
<p><span class="math inline">\(+\)</span> internal tRNA fragments &amp; piRNAs</p>
</section><section id="trna-expression" class="slide level2">
<h2>tRNA expression</h2>
<ul>
<li>tRNA genes act as insulators <small>(Raab et al.¬†2011 <em>EMBO</em> <span class="citation" data-cites="Raab2011">[<a href="#/ref-Raab2011" role="doc-biblioref">5</a>]</span>)</small></li>
<li>Clustered with other tRNA genes in 3D <small>(VanBortle et al.¬†2017 <em>Genome Biol</em> <span class="citation" data-cites="VanBortle2017">[<a href="#/ref-VanBortle2017" role="doc-biblioref">6</a>]</span>)</small></li>
<li>RNA polymerase III <small>(Schramm &amp; Hernandez 2002 <em>Genes &amp; Dev</em> <span class="citation" data-cites="Schramm2002">[<a href="#/ref-Schramm2002" role="doc-biblioref">7</a>]</span>)</small></li>
<li>Internal Promoter Structure</li>
<li>Experimentally DNAm <span class="math inline">\(\uparrow\ \rightarrow\)</span> tRNA Expression <span class="math inline">\(\downarrow\)</span> <small>Besser et al.¬†1990 <em>FEBS letters</em> <span class="citation" data-cites="Besser1990">[<a href="#/ref-Besser1990" role="doc-biblioref">8</a>]</span></small></li>
</ul>
<p><img src="graphics/tRNA-PollIII_white.png" width="50%" style="display: block; margin: auto;" /></p>
</section><section id="trnas-in-ageing" class="slide level2">
<h2>tRNAs in Ageing</h2>
<ul>
<li><span class="math inline">\(\downarrow\)</span> Translation, <span class="math inline">\(\uparrow\)</span> Longevity <small>(Hansen et al.¬†2007 <em>Aging Cell</em><span class="citation" data-cites="Hansen2007">[<a href="#/ref-Hansen2007" role="doc-biblioref">9</a>]</span>)</small></li>
<li>Ageing ~ TORC1 <span class="math inline">\(\dashv\)</span> Maf1 <span class="math inline">\(\dashv\)</span> polIII <span class="math inline">\(\dashv\)</span> tRNA expression</li>
<li><span class="math inline">\(\downarrow\)</span> polIII, <span class="math inline">\(\uparrow\)</span> Longevity <small>(Filer et al.¬†2017 <em>Nature</em> <span class="citation" data-cites="Filer2017">[<a href="#/ref-Filer2017" role="doc-biblioref">10</a>]</span>)</small></li>
<li>Circulating tRNA halves modulated by ageing and calorie restriction <small>(Dhahbi et al.¬†2013 <em>BMC Genomics</em><span class="citation" data-cites="Dhahbi2013">[<a href="#/ref-Dhahbi2013" role="doc-biblioref">11</a>]</span>)</small></li>
<li>tsRNAs in stress response &amp; Translational regulation</li>
<li><strong>tRNA biology is intimately connected with many systems which modulate ageing</strong></li>
</ul>
</section><section id="human-trnaome" class="slide level2">
<h2>Human tRNAome</h2>
<ul>
<li>Up to 610 tRNA genes <small>Chan 2009 <em>NAR</em> <span class="citation" data-cites="Chan2009">[<a href="#/ref-Chan2009" role="doc-biblioref">12</a>]</span></small></li>
<li>median length 73bp</li>
<li>&lt;46kb, 0.002% of the Genome</li>
<li>2nd most abundant RNA species, ~60 million/cell</li>
</ul>
<p><br></p>
<div id="htmlwidget-c62b405a7b7aebdf8628" style="width:800px;height:400px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-c62b405a7b7aebdf8628">{"x":{"visdat":{"55315a162a83":["function () ","plotlyVisDat"]},"cur_data":"55315a162a83","attrs":{"55315a162a83":{"labels":{},"values":{},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"pie","hole":0.6,"inherit":true}},"layout":{"width":800,"height":400,"margin":{"b":40,"l":60,"t":50,"r":10},"title":"Proportion of tRNAs By Chromosome","showlegend":true,"xaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"yaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"hovermode":"closest"},"source":"A","config":{"showSendToCloud":false},"data":[{"labels":["chr1","chr1_gl000192_random","chr10","chr11","chr12","chr13","chr14","chr15","chr16","chr17","chr18","chr19","chr2","chr20","chr21","chr22","chr3","chr4","chr5","chr6","chr7","chr8","chr9","chrX"],"values":[127,2,7,21,15,5,24,12,32,42,1,14,26,1,2,1,9,5,25,176,32,11,10,10],"type":"pie","hole":0.6,"marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(255,255,255,1)"}},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<section data-notes="
    - 23 nuclear encoded mitochondrial tRNAs
    - 116 potential pseudogenes
    - 107 are in blacklisted regions
    - 7 of the 61 non-STOP codons are missing - wobble pairing
        - (*e.g.* GCG Arg, ACC Gly)
">
</section>
</section></section>
<section><section id="results" class="title-slide slide level1"><h1>Results</h1></section><section id="twins-uk-medip-seq-data" class="slide level2">
<h2>Twins UK MeDIP-seq Data</h2>
<ul>
<li>4350 DNA methylomes (Whole Blood)</li>
<li>3001 with Cell-type counts</li>
<li>16 - 82 yrs</li>
<li>Sequencing based technology
<ul>
<li>50bp reads, 200-500bp fragments</li>
<li>Good Coverage but limited positional resolution</li>
</ul></li>
</ul>
<p><small>Bell et al.¬†2017 <em>Nat Comm</em> <span class="citation" data-cites="Bell2017a">[<a href="#/ref-Bell2017a" role="doc-biblioref">13</a>]</span> </small></p>
<p><img src="graphics/chip-seq.jpeg" width="35%" style="display: block; margin: auto;" /><img src="graphics/500bpwindowsing_bg.png" width="35%" style="display: block; margin: auto;" /></p>
<section data-notes="
    - Quantile normalised RPM
">
</section>
</section><section id="age-related-dna-hypermethyation" class="slide level2">
<h2>Age-related DNA hypermethyation</h2>
<ul>
<li>tRNAome-wide significant (p &lt; <span class="math inline">\(8.36\times10^{-5}\)</span>)
<ul>
<li>Batch corrected
<ul>
<li><strong>44</strong> <span class="math inline">\(\uparrow\)</span>, <span class="math inline">\(\downarrow\)</span> 0</li>
</ul></li>
<li>Batch and Blood Cell-type corrected
<ul>
<li><strong>16</strong> <span class="math inline">\(\uparrow\)</span>, <span class="math inline">\(\downarrow\)</span> 0</li>
</ul></li>
</ul></li>
<li>Genome-wide significant (p &lt; <span class="math inline">\(4.34\times10^{-9}\)</span>)
<ul>
<li>3 Batch and Blood Cell-type corrected</li>
</ul></li>
</ul>
<p><br></p>
<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
tRNA
</th>
<th style="text-align:left;">
Location
</th>
<th style="text-align:left;">
MeDIP-seq p-value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;font-weight: bold;border-right:1px solid;">
tRNA-iMet-CAT-1-4
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:left;">
2.83e-11
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;border-right:1px solid;">
tRNA-Ile-AAT-4-1
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:left;">
3.03e-10
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;border-right:1px solid;">
tRNA-Ser-AGA-2-6
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:left;">
1.16e-10
</td>
</tr>
</tbody>
</table>
<section data-notes="
    - Ill-AAT-4-1 &amp; Ser-AGA-2-6 3&#39; UTR of CTC1 (CST Telomere Replication Complex Component 1)
">
</section>
</section><section id="enrichment-for-dna-hypermethylation" class="slide level2">
<h2>Enrichment for DNA hypermethylation</h2>
<p><img src="graphics/nSigWinProps_WGfishers_BB_sws_hyper_2_BL_noPolyBivExcl_BarTxtast_sr.png" width="80%" style="display: block; margin: auto;" /></p>
</section><section id="medip-seq-limitations" class="slide level2">
<h2>MeDIP-seq limitations</h2>
<ul>
<li>Regional
<ul>
<li>Good Coverage but limited positional resolution</li>
</ul></li>
<li>Imperfect methylation Quantitation (especially at low DNAm)</li>
</ul>
<p><img src="graphics/chip-seq.jpeg" width="25%" style="display: block; margin: auto;" /></p>
</section><section id="neonate-centenarian" class="slide level2">
<h2>Neonate &amp; Centenarian</h2>
<ul>
<li>Whole genome Bisulfite Sequencing (WGBS)</li>
<li>tRNA hypermethylation even between single individuals</li>
</ul>
<p><img src="graphics/heynViolinBBswsOnlyLabelled_BL.png" width="45%" style="display: block; margin: auto;" /></p>
</section><section id="validation-in-450k-array" class="slide level2">
<h2>Validation in 450k array</h2>
<ul>
<li>Good Methylation Quantitation</li>
<li>CpG specific, probes within tRNA genes</li>
<li>Covers 103 tRNAs, 7 of the 16 tRNAome significant</li>
<li>5 of these show an increase (<span class="math inline">\(4.39\times10^{-4}\)</span>, N = 587)</li>
<li>Cell-type corrected</li>
</ul>
<p><img src="graphics/blood_epitwins450k_Counts_sws_bl.png" width="50%" style="display: block; margin: auto;" /></p>
</section><section id="isodecoders" class="slide level2">
<h2>Isodecoders</h2>
<ul>
<li>tRNA with the same anticodon but different sequence elsewhere</li>
<li>Numbers increase with organismal complexity <small>(Goodenbour et al.¬†2006 <em>NAR</em> <span class="citation" data-cites="Goodenbour2006">[<a href="#/ref-Goodenbour2006" role="doc-biblioref">14</a>]</span>)</small></li>
<li>Non-equivalent translational efficiency <small>(Geslain et al.¬†2010 <em>J Mol Biol</em> <span class="citation" data-cites="Geslain2010">[<a href="#/ref-Geslain2010" role="doc-biblioref">15</a>]</span>)</small></li>
</ul>
<p><img src="graphics/isodecoder.png" width="20%" style="display: block; margin: auto;" /></p>
</section><section id="targeted-bisulfite-sequencing-bis-seq" class="slide level2">
<h2>Targeted Bisulfite Sequencing (BiS-seq)</h2>
<p>Pooled Samples approach (N = 190, ~25 per pool)</p>
<p><img src="graphics/pooling.png" width="55%" style="display: block; margin: auto;" /></p>
</section><section id="targeted-bis-seq-results" class="slide level2">
<h2>Targeted (BiS-seq) Results</h2>
<ul>
<li>79 tRNA loci</li>
<li>Differences between Duplicates / Isodecoders</li>
</ul>
<p><img src="graphics/combinedKaryoplots_sig_bl_1.png" width="85%" style="display: block; margin: auto;" /></p>
</section><section id="mouse" class="slide level2">
<h2>Mouse üêÅ</h2>
<ul>
<li>Reduced Representation Bisulfite Sequencing (RRBS)</li>
<li>N = 152, Aged 0.67-35 months</li>
<li>Covering 51 tRNAs (~11% of total)</li>
<li>3 significant all <span class="math inline">\(\uparrow\)</span> (p &lt; <span class="math inline">\(1.08 \times 10^{-4}\)</span>)</li>
</ul>
<p><img src="graphics/CpGPlotComb.png" width="60%" style="display: block; margin: auto;" /></p>
<p><small>Petkovich et al.¬†2017 <em>Cell Met</em> <span class="citation" data-cites="Petkovich2017">[<a href="#/ref-Petkovich2017" role="doc-biblioref">16</a>]</span></small></p>
</section></section>
<section><section id="functional-consequences" class="title-slide slide level1"><h1>Functional Consequences</h1></section><section id="chromatin-state" class="slide level2">
<h2>Chromatin State</h2>
<p>Age hypermethylating tRNAs more Enhancer-like state (Enh, EnhBiv &amp; EnhG) - (Fisher‚Äôs Exact test p = 0.01)</p>
<p><img src="graphics/maxScoreBytRNAtRNAge6SplitPlot_BL.png" width="65%" style="display: block; margin: auto;" /></p>
<p><a href="https://epilogos.altius.org/">Epilogos</a> ‚ÄòBlood &amp; T-cell‚Äô 15 State model (tRNA genes +/- 200bp)</p>
</section></section>
<section id="conclusions-discussion" class="title-slide slide level1"><h1>Conclusions &amp; Discussion</h1><ul>
<li>The Human tRNAome is enriched for age related DNA hypermethylation
<ul>
<li>Some blood cell-type independent</li>
<li>Not driven by CpG density</li>
</ul></li>
<li>Validated select tRNAs in 450k array</li>
<li>Replicated select tRNAs in Targeted BiS-seq</li>
<li>Evidence of tRNA gene hypermethylation in mouse</li>
<li>tRNA gene DNAm differences between blood cell-lineages</li>
</ul>
<p><span class="math inline">\(BioR \chi v\)</span> <a href="https://doi.org/10.1101/870352">https://doi.org/10.1101/870352</a> - Under Consideration at Nature Communications</p></section>
<section><section id="other-work" class="title-slide slide level1"><h1>Other Work</h1></section><section id="haplotype-specific-methylation-website" class="slide level2">
<h2>Haplotype Specific Methylation Website</h2>
<p><img src="graphics/HSM-website.png" width="70%" style="display: block; margin: auto;" /></p>
</section><section id="ewas" class="slide level2">
<h2>EWAS</h2>
<p>E.g. Periosteal circumference (6yr)</p>
<p><img src="graphics/s3peric-EWAS-all-SWS-manhattan.png" width="80%" style="display: block; margin: auto;" /></p>
</section></section>
<section id="current-work---alu-clock" class="title-slide slide level1"><h1>Current Work - Alu Clock</h1><ul>
<li>Constructing a DNAm clock using Alu elements</li>
<li>MeDIP-seq data coverage of regions not on the array</li>
<li>Compare to other clocks</li>
<li>GWAS for Alu Age acceleration</li>
</ul>
<p><img src="graphics/Alu_study_design_2019-12-05.png" width="85%" style="display: block; margin: auto;" /></p></section>
<section><section id="acknowledgements" class="title-slide slide level1"><h1>Acknowledgements</h1><p><img src="graphics/KCL_white_logo.png" width="144" height="10%" /><img src="graphics/MRC_LEU_logo.png" width="279" height="10%" /><img src="graphics/QMC_barts_logo.png" width="291" height="10%" /><img src="graphics/soton_white_logo.png" width="327" height="10%" /></p>
<ul>
<li><strong>Supervisors:</strong> Christopher G. Bell, Karen Lillicrop, Cyrus Cooper</li>
<li><strong>MRC LEU:</strong> Elaine Dennison (Hertfordshire), Cyrus Cooper, Nicholas Harvey (MAVIDOS), Nikki Graham</li>
<li><strong>Barts and the London:</strong> Charles Mein, Emma Bourne, Eva Wozniak</li>
<li><strong>TwinsUK / KCL:</strong> Tim Spector, Pirro Hysi, Jordana Bell, Wei Yuan</li>
<li><strong>BGI shenzen:</strong> Fei Gao, Yudong Xia, Jun Wang</li>
<li>IRIDIS High-Performance Computing Facility</li>
<li><strong>Funding:</strong> Medical Research Council</li>
</ul></section><section id="slides" class="slide level2">
<h2>Slides</h2>
<p>Contact: <a href="mailto:r.j.acton@soton.ac.uk">r.j.acton@soton.ac.uk</a></p>
<p>github: <a href="http://richardjacton.github.io">richardjacton.github.io</a></p>
<p>orcid: <a href="https://orcid.org/0000-0002-2574-9611">0000-0002-2574-9611</a></p>
<p><img src="tRNA-presentation-2019-12-06_files/figure-revealjs/unnamed-chunk-25-1.png" style="display: block; margin: auto;" /></p>
</section></section>
<section><section id="references" class="title-slide slide level1"><h1>References</h1></section><section class="slide level2">

<small><small><small>
<div id="refs" role="doc-bibliography">
<div id="ref-Booth2016">
<p>1. Booth LN, Brunet A: <strong>The Aging Epigenome</strong>. <em>Molecular Cell</em> 2016, <strong>62</strong>:728‚Äì744<a href="https://doi.org/10.1016/j.molcel.2016.05.013">10.1016/j.molcel.2016.05.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2016.05.013">http://dx.doi.org/10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div id="ref-Horvath2013">
<p>2. Horvath S: <strong>DNA methylation age of human tissues and cell types.</strong> <em>Genome biology</em> 2013, <strong>14</strong>:R115<a href="https://doi.org/10.1186/gb-2013-14-10-r115">10.1186/gb-2013-14-10-r115</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24138928">http://www.ncbi.nlm.nih.gov/pubmed/24138928</a>.</p>
</div>
<div id="ref-Marioni2015">
<p>3. Marioni RE, Shah S, McRae AF, Chen BH, Colicino E, Harris SE, Gibson J, Henders AK, Redmond P, Cox SR, Pattie A, Corley J, Murphy L, Martin NG, Montgomery GW, Feinberg AP, Fallin MD, Multhaup ML, Jaffe AE, Joehanes R, Schwartz J, Just AC, Lunetta KL, Murabito JM, Starr JM, Horvath S, Baccarelli AA, Levy D, Visscher PM, Wray NR, et al.: <strong>DNA methylation age of blood predicts all-cause mortality in later life.</strong> <em>Genome biology</em> 2015, <strong>16</strong>:25<a href="https://doi.org/10.1186/s13059-015-0584-6">10.1186/s13059-015-0584-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25633388">http://www.ncbi.nlm.nih.gov/pubmed/25633388</a>.</p>
</div>
<div id="ref-Bell2016">
<p>4. Bell CG, Xia Y, Yuan W, Gao F, Ward K, Roos L, Mangino M, Hysi PG, Bell J, Wang J, Spector TD: <strong>Novel regional age-associated DNA methylation changes within human common disease-associated loci</strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:193<a href="https://doi.org/10.1186/s13059-016-1051-8">10.1186/s13059-016-1051-8</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27663977">http://www.ncbi.nlm.nih.gov/pubmed/27663977</a>.</p>
</div>
<div id="ref-Raab2011">
<p>5. Raab JR, Chiu J, Zhu J, Katzman S, Kurukuti S, Wade PA, Haussler D, Kamakaka RT: <strong>Human tRNA genes function as chromatin insulators.</strong> <em>The EMBO journal</em> 2012, <strong>31</strong>:330‚Äì50<a href="https://doi.org/10.1038/emboj.2011.406">10.1038/emboj.2011.406</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22085927">http://www.ncbi.nlm.nih.gov/pubmed/22085927</a>.</p>
</div>
<div id="ref-VanBortle2017">
<p>6. Van Bortle K, Phanstiel DH, Snyder MP: <strong>Topological organization and dynamic regulation of human tRNA genes during macrophage differentiation</strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:180<a href="https://doi.org/10.1186/s13059-017-1310-3">10.1186/s13059-017-1310-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1310-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1310-3</a>.</p>
</div>
<div id="ref-Schramm2002">
<p>7. Schramm L: <strong>Recruitment of RNA polymerase III to its target promoters</strong>. <em>Genes &amp; Development</em> 2002, <strong>16</strong>:2593‚Äì2620<a href="https://doi.org/10.1101/gad.1018902">10.1101/gad.1018902</a>Available: <a href="http://www.genesdev.org/cgi/doi/10.1101/gad.1018902">http://www.genesdev.org/cgi/doi/10.1101/gad.1018902</a>.</p>
</div>
<div id="ref-Besser1990">
<p>8. Besser D, G√∂tz F, Schulze-Forster K, Wagner H, Kr√∂ger H, Simon D: <strong>DNA methylation inhibits transcription by RNA polymerase III of a tRNA gene, but not of a 5S rRNA gene.</strong> <em>FEBS letters</em> 1990, <strong>269</strong>:358‚Äì62<a href="https://doi.org/10.1016/0014-5793(90)81193-R">10.1016/0014-5793(90)81193-R</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/2401361">http://www.ncbi.nlm.nih.gov/pubmed/2401361</a>.</p>
</div>
<div id="ref-Hansen2007">
<p>9. Hansen M, Taubert S, Crawford D, Libina N, Lee S-J, Kenyon C: <strong>Lifespan extension by conditions that inhibit translation in Caenorhabditis elegans</strong>. <em>Aging Cell</em> 2007, <strong>6</strong>:95‚Äì110<a href="https://doi.org/10.1111/j.1474-9726.2006.00267.x">10.1111/j.1474-9726.2006.00267.x</a>Available: <a href="http://doi.wiley.com/10.1111/j.1474-9726.2006.00267.x">http://doi.wiley.com/10.1111/j.1474-9726.2006.00267.x</a>.</p>
</div>
<div id="ref-Filer2017">
<p>10. Filer D, Thompson MA, Takhaveev V, Dobson AJ, Kotronaki I, Green JWM, Heinemann M, Tullet JMA, Alic N: <strong>RNA polymerase III limits longevity downstream of TORC1</strong>. <em>Nature</em> 2017, <strong>552</strong>:263‚Äì267<a href="https://doi.org/10.1038/nature25007">10.1038/nature25007</a>Available: <a href="http://www.nature.com/articles/nature25007">http://www.nature.com/articles/nature25007</a>.</p>
</div>
<div id="ref-Dhahbi2013">
<p>11. Dhahbi JM, Spindler SR, Atamna H, Yamakawa A, Boffelli D, Mote P, Martin DI: <strong>5‚Äô tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction</strong>. <em>BMC Genomics</em> 2013, <strong>14</strong>:298<a href="https://doi.org/10.1186/1471-2164-14-298">10.1186/1471-2164-14-298</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23638709">http://www.ncbi.nlm.nih.gov/pubmed/23638709</a>.</p>
</div>
<div id="ref-Chan2009">
<p>12. Chan PP, Lowe TM: <strong>GtRNAdb: a database of transfer RNA genes detected in genomic sequence.</strong> <em>Nucleic acids research</em> 2009, <strong>37</strong>:D93‚Äì7<a href="https://doi.org/10.1093/nar/gkn787">10.1093/nar/gkn787</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/18984615">http://www.ncbi.nlm.nih.gov/pubmed/18984615</a>.</p>
</div>
<div id="ref-Bell2017a">
<p>13. Bell CG, Gao F, Yuan W, Roos L, Acton RJ, Xia Y, Bell J, Ward K, Mangino M, Hysi PG, Wang J, Spector TD: <strong>Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci</strong>. <em>Nature Communications</em> 2018, <strong>9</strong>:8<a href="https://doi.org/10.1038/s41467-017-01586-1">10.1038/s41467-017-01586-1</a>Available: <a href="http://www.nature.com/articles/s41467-017-01586-1">http://www.nature.com/articles/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Goodenbour2006">
<p>14. Goodenbour JM, Pan T: <strong>Diversity of tRNA genes in eukaryotes.</strong> <em>Nucleic acids research</em> 2006, <strong>34</strong>:6137‚Äì46<a href="https://doi.org/10.1093/nar/gkl725">10.1093/nar/gkl725</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17088292">http://www.ncbi.nlm.nih.gov/pubmed/17088292</a>.</p>
</div>
<div id="ref-Geslain2010">
<p>15. Geslain R, Pan T: <strong>Functional Analysis of Human tRNA Isodecoders</strong>. <em>Journal of Molecular Biology</em> 2010, <strong>396</strong>:821‚Äì831<a href="https://doi.org/10.1016/j.jmb.2009.12.018">10.1016/j.jmb.2009.12.018</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S002228360901523X">https://linkinghub.elsevier.com/retrieve/pii/S002228360901523X</a>.</p>
</div>
<div id="ref-Petkovich2017">
<p>16. Petkovich DA, Podolskiy DI, Lobanov AV, Lee S-G, Miller RA, Gladyshev VN: <strong>Using DNA Methylation Profiling to Evaluate Biological Age and Longevity Interventions</strong>. <em>Cell Metabolism</em> 2017, <strong>25</strong>:954‚Äì960.e6<a href="https://doi.org/10.1016/j.cmet.2017.03.016">10.1016/j.cmet.2017.03.016</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1550413117301687">https://linkinghub.elsevier.com/retrieve/pii/S1550413117301687</a>.</p>
</div>
<div id="ref-Schimmel2017">
<p>17. Schimmel P: <strong>The emerging complexity of the tRNA world: mammalian tRNAs beyond protein synthesis.</strong> <em>Nature reviews. Molecular cell biology</em> 2018, <strong>19</strong>:45‚Äì58<a href="https://doi.org/10.1038/nrm.2017.77">10.1038/nrm.2017.77</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28875994">http://www.ncbi.nlm.nih.gov/pubmed/28875994</a>.</p>
</div>
<div id="ref-Kirchner2017">
<p>18. Kirchner S, Cai Z, Rauscher R, Kastelic N, Anding M, Czech A, Kleizen B, Ostedgaard LS, Braakman I, Sheppard DN, Ignatova Z: <strong>Alteration of protein function by a silent polymorphism linked to tRNA abundance</strong>. <em>PLOS Biology</em> 2017, <strong>15</strong>:e2000779<a href="https://doi.org/10.1371/journal.pbio.2000779">10.1371/journal.pbio.2000779</a>Available: <a href="http://dx.plos.org/10.1371/journal.pbio.2000779">http://dx.plos.org/10.1371/journal.pbio.2000779</a>.</p>
</div>
<div id="ref-Blanco2014">
<p>19. Blanco S, Dietmann S, Flores JV, Hussain S, Kutter C, Humphreys P, Lukk M, Lombard P, Treps L, Popis M, Kellner S, H√∂lter SM, Garrett L, Wurst W, Becker L, Klopstock T, Fuchs H, Gailus‚ÄêDurner V, Hrabƒï de Angelis M, K√°rad√≥ttir RT, Helm M, Ule J, Gleeson JG, Odom DT, Frye M: <strong> Aberrant methylation of t RNA s links cellular stress to neuro‚Äêdevelopmental disorders</strong>. <em>The EMBO Journal</em> 2014, <strong>33</strong>:2020‚Äì2039<a href="https://doi.org/10.15252/embj.201489282">10.15252/embj.201489282</a>.</p>
</div>
<div id="ref-Reinius2012">
<p>20. Reinius LE, Acevedo N, Joerink M, Pershagen G, Dahl√©n S-E, Greco D, S√∂derh√§ll C, Scheynius A, Kere J: <strong>Differential DNA Methylation in Purified Human Blood Cells: Implications for Cell Lineage and Studies on Disease Susceptibility</strong>. <em>PLoS ONE</em> 2012, <strong>7</strong>:e41361<a href="https://doi.org/10.1371/journal.pone.0041361">10.1371/journal.pone.0041361</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0041361">https://dx.plos.org/10.1371/journal.pone.0041361</a>.</p>
</div>
<div id="ref-Loher2017">
<p>21. Loher P, Telonis AG, Rigoutsos I: <strong>MINTmap: fast and exhaustive profiling of nuclear and mitochondrial tRNA fragments from short RNA-seq data.</strong> <em>Scientific reports</em> 2017, <strong>7</strong>:41184<a href="https://doi.org/10.1038/srep41184">10.1038/srep41184</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28220888">http://www.ncbi.nlm.nih.gov/pubmed/28220888</a>.</p>
</div>
<div id="ref-Juzenas2017">
<p>22. Juzenas S, Venkatesh G, H√ºbenthal M, Hoeppner MP, Du ZG, Paulsen M, Rosenstiel P, Senger P, Hofmann-Apitius M, Keller A, Kupcinskas L, Franke A, Hemmrich-Stanisak G: <strong>A comprehensive, cell specific microRNA catalogue of human peripheral blood</strong>. <em>Nucleic Acids Research</em> 2017, <strong>45</strong>:9290‚Äì9301<a href="https://doi.org/10.1093/nar/gkx706">10.1093/nar/gkx706</a>Available: <a href="http://academic.oup.com/nar/article/45/16/9290/4080663">http://academic.oup.com/nar/article/45/16/9290/4080663</a>.</p>
</div>
<div id="ref-Muller2017">
<p>23. M√ºller CA, Nieduszynski CA: <strong>DNA replication timing influences gene expression level</strong>. <em>The Journal of Cell Biology</em> 2017, <strong>216</strong>:1907‚Äì1914<a href="https://doi.org/10.1083/jcb.201701061">10.1083/jcb.201701061</a>Available: <a href="http://www.jcb.org/lookup/doi/10.1083/jcb.201701061">http://www.jcb.org/lookup/doi/10.1083/jcb.201701061</a>.</p>
</div>
</div>
<p></small></small></small></p>
</section><section id="human-trnaome-1" class="slide level2">
<h2>Human tRNAome</h2>
<p><img src="graphics/kutteresq_BLcount.png" width="85%" style="display: block; margin: auto;" /></p>
<p>() = pseudo genes, [] = in blacklisted regions</p>
</section><section id="trna-modifications" class="slide level2">
<h2>tRNA modifications</h2>
<ul>
<li>Most heavily modified RNA species
<ul>
<li><span class="math inline">\(&gt;\)</span> 100 different modifications</li>
</ul></li>
<li>11-13 per tRNA molecule <small>(Schimmel 2017 <em>Nat Rev Mol Cell Biol</em> <span class="citation" data-cites="Schimmel2017">[<a href="#/ref-Schimmel2017" role="doc-biblioref">17</a>]</span>)</small></li>
</ul>
<p><img src="graphics/tRNA_modifications_yeast.jpeg" width="55%" style="display: block; margin: auto;" /></p>
</section><section id="trnas-disease" class="slide level2">
<h2>tRNAs &amp; Disease</h2>
<ul>
<li>Synonymous CFTR mutation to low abundance tRNA causes misfolding <small>(Kirchner et al.¬†2017 <em>PLoS Bio</em> <span class="citation" data-cites="Kirchner2017">[<a href="#/ref-Kirchner2017" role="doc-biblioref">18</a>]</span>)</small></li>
<li>tRNA fragments implicated in neuro-developmental disorders <small>(Blanco et al.¬†2014 <em>EMBO</em> <span class="citation" data-cites="Blanco2014">[<a href="#/ref-Blanco2014" role="doc-biblioref">19</a>]</span>)</small></li>
</ul>
</section><section id="mappability" class="slide level2">
<h2>Mappability</h2>
<p><img src="graphics/mappabilityScoreDensity_tRNAVsFlank500.png" width="65%" style="display: block; margin: auto;" /></p>
</section><section id="dnamage-of-pooled-samples" class="slide level2">
<h2>DNAmAge of Pooled Samples</h2>
<p>DNAm age prediction to test pooling approach</p>
<p><img src="graphics/DNAmAgeVsChronAge.png" width="55%" style="display: block; margin: auto;" /></p>
<p><small>Horvath 2013 <em>Genome Biol</em> <span class="citation" data-cites="Horvath2013">[<a href="#/ref-Horvath2013" role="doc-biblioref">2</a>]</span></small></p>
</section><section id="cpg-density-permutation" class="slide level2">
<h2>CpG Density Permutation</h2>
<ul>
<li>Sampled windows with matching CpG density</li>
</ul>
<p><img src="graphics/tRNA-poly-bivP_n1000_combined_perm_hist.png" width="75%" style="display: block; margin: auto;" /></p>
</section><section id="trna-imet-cat-1-4-individual-cpg-hypermethylation" class="slide level2">
<h2>tRNA-iMet-CAT-1-4 individual CpG Hypermethylation</h2>
<p><img src="graphics/tRNA-iMet-CAT-1-4_nomSig_CpG.png" width="60%" style="display: block; margin: auto;" /></p>
</section><section id="trna-gene-copy-number" class="slide level2">
<h2>tRNA gene Copy Number</h2>
<p><img src="graphics/kutteresq_BLcount.png" width="85%" style="display: block; margin: auto;" /></p>
<p>() = pseudo genes, [] = in blacklisted regions</p>
</section><section id="trna-gene-methylation-by-cell-type" class="slide level2">
<h2>tRNA gene methylation by Cell-type</h2>
<ul>
<li>Sorted blood cell populations, 450k array <small>Reinius et al.¬†2012 <em>PLoS one</em> <span class="citation" data-cites="Reinius2012">[<a href="#/ref-Reinius2012" role="doc-biblioref">20</a>]</span></small></li>
<li>8 study-wide significant tRNAs covered</li>
<li>Collectively more methylated in lymphoid than myeloid lineage
<ul>
<li>(1.1% difference, Wilcoxon rank sum test p = <span class="math inline">\(1.50\times10^{-6}\)</span>)</li>
</ul></li>
</ul>
<p><span class="math inline">\(\therefore\)</span></p>
<p>Age-related <span class="math inline">\(\uparrow\)</span> myeloid cell proportion <span class="math inline">\(\rightarrow\)</span> age-related hypermethylation <span class="math inline">\(\downarrow\)</span></p>
<p><img src="graphics/meanMethbytRNAMatLowBB6_bl.png" width="55%" style="display: block; margin: auto;" /></p>
</section><section id="expression" class="slide level2">
<h2>Expression</h2>
</section><section id="mintmap" class="slide level2">
<h2>MINTmap</h2>
<ul>
<li>Exhaustive exact search of tRNA sequence space</li>
<li>Small RNA-seq (read length = 50nt)</li>
<li>combined data from Sorted blood cell types</li>
<li>N = 42, aged 21 - 63 yrs</li>
<li>Fragment length &gt;40nt</li>
<li>5384 unique fragments derived from as many as 417 tRNA loci</li>
<li><strong>Many-to-many tsRNA to tRNA gene mapping</strong></li>
</ul>
<p><small> Loher et al.¬†2017 <em>Sci Repts</em> <span class="citation" data-cites="Loher2017">[<a href="#/ref-Loher2017" role="doc-biblioref">21</a>]</span>, Juzenas et al.¬†2017 <em>NAR</em> <span class="citation" data-cites="Juzenas2017">[<a href="#/ref-Juzenas2017" role="doc-biblioref">22</a>]</span> </small></p>
</section><section id="mintmap-results" class="slide level2">
<h2>MINTmap Results</h2>
<ul>
<li>48 tsRNAs with nominally significant expression changes (p &lt; 0.05)
<ul>
<li>8 <span class="math inline">\(\uparrow\)</span>, 40 <span class="math inline">\(\downarrow\)</span> abundance with age</li>
</ul></li>
<li>tRNA-Gln-CTG-7-1 is in the 16 tRNAome-wide significant hypermethylators</li>
</ul>
<p><img src="graphics/nonSigIsoMapbbswsPlotsGln-CTG.png" width="65%" style="display: block; margin: auto;" /></p>
<ul>
<li>Suggests age-related decline in Gln-CTG family tRNA expression</li>
</ul>
</section><section class="slide level2">

<p>structure notes</p>
<ul>
<li>Why?
<ul>
<li>Ageing</li>
<li>epigenetics ~ ageing</li>
<li>prior tRNA result ~ artifact?</li>
<li>tRNA biology ~ ageing
<ul>
<li>tsRNAs</li>
<li>tRNA modifications</li>
</ul></li>
</ul></li>
<li>What - Results
<ul>
<li>meDip
<ul>
<li>immunoprecipitation / regional</li>
</ul></li>
<li>age related DNA hypermethylation</li>
<li>enrichment, CpG density permutation</li>
<li>array
<ul>
<li>bisulfite conversion based, highly quantitative</li>
<li>within the tRNA</li>
</ul></li>
<li>Targeted
<ul>
<li>pooling</li>
<li>compare duplicates / isodecoders</li>
</ul></li>
</ul></li>
<li>tRNA are in early replicating regions
<ul>
<li>Possible reason for hypermethylation <small>(Muller &amp; Nieduszynski 2017 <em>J Cell Biol</em>, <span class="citation" data-cites="Muller2017">[<a href="#/ref-Muller2017" role="doc-biblioref">23</a>]</span>, Du et at. 2019 <em>Nat Comms</em> <span class="citation" data-cites="Du1029">[<span class="citeproc-not-found" data-reference-id="Du1029"><strong>???</strong></span>]</span>)</small></li>
</ul></li>
</ul>
</section></section>
    </div>
  </div>

  <script src="tRNA-presentation-2019-12-06_files/reveal.js-3.3.0.1/lib/js/head.min.js"></script>
  <script src="tRNA-presentation-2019-12-06_files/reveal.js-3.3.0.1/js/reveal.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://github.com/hakimel/reveal.js#configuration
      Reveal.initialize({
        // Display the page number of the current slide
        slideNumber: true,
        // Push each slide change to the browser history
        history: true,
        // Vertical centering of slides
        center: true,
        // Opens links in an iframe preview overlay
        previewLinks: true,
        // Transition style
        transition: 'fade', // none/fade/slide/convex/concave/zoom
        // Transition style for full page slide backgrounds
        backgroundTransition: 'default', // none/fade/slide/convex/concave/zoom



        // Optional reveal.js plugins
        dependencies: [
          { src: 'tRNA-presentation-2019-12-06_files/reveal.js-3.3.0.1/plugin/notes/notes.js', async: true },
        ]
      });
    </script>
  <!-- dynamically load mathjax for compatibility with self-contained -->
  <script>
    (function () {
      var script = document.createElement("script");
      script.type = "text/javascript";
      script.src  = "tRNA-presentation-2019-12-06_files/mathjax-local/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
      document.getElementsByTagName("head")[0].appendChild(script);
    })();
  </script>

<script>
  (function() {
    if (window.jQuery) {
      Reveal.addEventListener( 'slidechanged', function(event) {  
        window.jQuery(event.previousSlide).trigger('hidden');
        window.jQuery(event.currentSlide).trigger('shown');
      });
    }
  })();
</script>


  </body>
</html>
